qwerty_pyrt/python/qwerty/repl.py

```diff
@@ -12,12 +12,14 @@
 import ast
 from .convert_ast import convert_qpu_repl_input
 from .err import QwertyProgrammerError
-from ._qwerty_pyrt import ReplState
+from ._qwerty_pyrt import ReplState, TypeEnv, Expr
```

Comment:
I don't think you need to import Expr, right?
```suggestion
from ._qwerty_pyrt import ReplState, TypeEnv
```